### PR TITLE
Add TypeScript declarations to paper-toggle-button.

### PR DESCRIPTION
**Pull request by aomarks**
This PR adds TypeScript declarations generated by https://github.com/Polymer/gen-typescript-declarations/

These declarations can be re-generated by running `npm run update-types`.

Tracker: https://github.com/Polymer/gen-typescript-declarations/issues/79

### CODE REVIEW

File: paper-toggle-button.d.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ * `--paper-toggle-button-checked-button` | Mixin applied to the slider button when the input is checked | `{}`
+ * `--paper-toggle-button-checked-ink` | Mixin applied to the ripple when the input is checked | `{}`
+ * `--paper-toggle-button-label-color` | Label color | `--primary-text-color`
+ * `--paper-toggle-button-label-spacing` | Spacing between the label and the button | `8px`
+ *
+ * This element applies the mixin `--paper-font-common-base` but does not import `paper-styles/typography.html`.
+ * In order to apply the `Roboto` font to this element, make sure you've imported `paper-styles/typography.html`.
+ */
+interface PaperToggleButtonElement extends Polymer.Element, Polymer.PaperCheckedElementBehavior {
+  hostAttributes: object|null;
```
Comment (bicknellr) [inline paper-toggle-button.d.ts]: Same as https://github.com/PolymerElements/paper-tabs/pull/222/files#r173251359.

**Review by dfreedm — APPROVED** [review]

File: paper-toggle-button.d.ts (hunk 1 above)
Comment (aomarks) [reply paper-toggle-button.d.ts]: Added `@private` annotation to remove.

**Comment by bicknellr** [comment]
I restarted your builds because one looked like a selenium issue and the other a sauce issue.

**Comment by aomarks** [comment]
> I restarted your builds because one looked like a selenium issue and the other a sauce issue.

Thanks! Passing now.

**Review by bicknellr — APPROVED** [review]

**Pull request merged**